src/components/Layout/FormPageLayout.tsx

```diff
@@ -28,17 +29,18 @@ type FormPageLayoutProps<F> = {
   submitAction?: ActionButtonProps;
   cancelAction?: ActionButtonProps;
   tertiaryAction?: ActionButtonProps;
+  rightSideBarContent?: SidebarContentProps[];
 };
 
 /** In order to make the multiple stacked sticky elements work (Header, then sidebar below) we need to set the header height.
  * The alternate solution was to have multiple defined grid areas for the fixed header vs body below, but this becomes more complex
  * to manage when adding in a max content width container for the page while keeping the page scrollbar to the far right of the page.
  * Rather than wrapping the page in a max-width div, we use "gutter" columns `minMax(0, auto)` that kick in when all other columns have met their max widths.
  */
-const headerHeightPx = 120;
+export const headerHeightPx = 120;
```

Comment:
nit: Is the export stale now? Hopefully this can stay as a local implementation detail

Reply:
good catch, I can remove the `export`